chsdi/static/doc/source/services/sdiservices.rst

```diff
@@ -754,21 +754,21 @@ The GetCapabilites document provides informations about the service, along with
 Parameters
 **********
 
-Only the RESTFul interface ist implemented. No KVP and SOAP. You *have* to provide a value for the `timestamp` parameter, the keywords `current` or 
+Only the RESTFul interface is implemented. No KVP and SOAP. You *have* to provide a value for the `timestamp` parameter, the keywords `current` or 
 `default` are not supported for now.
 
 A request is in the form:
 
 ::
 
-    <protocol>://<ServerName>/<ProtocoleVersion>/<LayerName>/<Stylename>/<Time>/<TileMatrixSet>/<TileSetId>/<TileRow>/<TileCol>.<FormatExtension>
+    <Scheme>://<ServerName>/<ProtocoleVersion>/<LayerName>/<Stylename>/<Time>/<TileMatrixSet>/<TileSetId>/<TileRow>/<TileCol>.<FormatExtension>
 
 with the following parameters:
 
 ===================    =============================   ==========================================================================
 Parameter              Example                         Explanation
 ===================    =============================   ==========================================================================
-Protocol               http ou https                   
+Scheme                 http or https                   The scheme type
```

Comment:
Ok then it's all good, this comment was actually under WMTS instead of Terrain Service.